backend/app/literature/tests/test_003_authors.py

```diff
@@ -41,7 +37,7 @@ def test_create_author():
         "first_name": "string",
         "last_name": "string",
         "name": "003_TCU",
-        "orcid": "BOB",
+        "orcid": "AUT:BOB",
```

Comment:
For what it's worth, the format for ORCIDs in the https://github.com/alliance-genome/agr_schemas/blob/master/resourceDescriptors.yaml is 
"^ORCID:\\d{4}\\-\\d{4}\\-\\d{4}\\-(\\d{3}X|\\d{4})$"  I don't know if it's helpful or unhelpful to have a more data-like test.

Reply:
There was no checking of format in the code so was unsure. But yes it is better to have something closer to the true values. I will update to something closer. Thanks.

Reply:
Good to know, thanks @ianlongden .  Yeah, there isn't any checking of format on the data coming in either, mostly it comes from PubMed, but if a DQM submitted something in the wrong format, it would go in that way.  Same with the curation UI as it currently stands.  Well, maybe the FMS validator prevents it going in if it's not the format from the https://github.com/alliance-genome/agr_schemas/blob/master/resourceDescriptors.yaml ?  I don't know how the validator works.  In the future we could add more restrictions.  The API example for ORCID also isn't explicit, just has the  ORCID prefix.  In general this data is not too important though, at least it won't be until we start trying to connect authors to Persons.